gateway-simulation/README.md

```diff
@@ -0,0 +1,52 @@
+# gwsim
+
+## Description
+
+Simulates a gateway supporting the server side of the [VDSL Performer](https://github.com/devolo/vdsl-performer-documentation/blob/main/README.md) protocol to obtain DSL SNR for DSL coexistence tests. devolo Magic devices utilize the TR-064 issue 2 standard to fetch data from a discovered DSL gateway, aiming to eliminate interference between PLC and DSL while maximizing the available throughput on PLC.
```

Comment:
Within Github links to other files can also be relative, if that would be preferable here.